### PR TITLE
st2-check-license added

### PR DESCRIPTION
**Pull request by punalpatel**
st2-check-license will verify whether license is valid or not

### CODE REVIEW

File: st2common/bin/st2-check-license [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+if [[ $# -eq 0 ]] ; then
+echo 'No License Key Entered'
```
Comment (lakshmi-kannan) [inline st2common/bin/st2-check-license]: Indent these two lines

File: st2common/bin/st2-check-license [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+echo 'Enter License Key'
+exit 0
+fi
+LICENSE=$@
+BWC_LICENSE_CHECK_URL="https://${LICENSE}:@packagecloud.io/install/repositories/StackStorm/enterprise/script.deb.sh"
+echo $LICENSE
```
Comment (lakshmi-kannan) [inline st2common/bin/st2-check-license]: Remove these two echo lines

File: st2common/bin/st2-check-license [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+echo $BWC_LICENSE_CHECK_URL
+LICENSE_CURL_TEST=`curl --output /dev/null --silent --fail ${BWC_LICENSE_CHECK_URL}`
+if [ $? -ne 0 ]; then
```
Comment (lakshmi-kannan) [inline st2common/bin/st2-check-license]: Instead of $?, shouldn't you be checking the value of variable LICENSE_CURL_TEST?

**Comment by Kami** [comment]
I agree with lakshmi - good overall idea.

Another thing which needs to be done when this is finished is packaging changes, but I can take over that if necessary.

**Comment by Kami** [comment]
Another thing - Python interpreter is required for StackStorm installation so if you feel more comfortable with Python I'm also fine with using Python instead of bash.

In any case, it's your call, I'm fine with both :)

File: st2common/bin/st2-check-license [hunk 4]
```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+if [[ $# -eq 0 ]] ; then
+   echo 'No License Key Entered'
+   echo 'Enter License Key'
```
Comment (Kami) [inline st2common/bin/st2-check-license]: Maybe better to say something like:

```bash
echo "No license key entered"
echo "Usage: $0 <license key to check>"
```

**Comment by lakshmi-kannan** [comment]
>Another thing which needs to be done when this is finished is packaging changes, but I can take over that if necessary.

@Kami This is the reason I asked him to put the file in st2common/bin/. Anything there is already packaged and put in /usr/bin/ I believe.

**Comment by Kami** [comment]
@lakshmi-kannan Yeah, you are correct - it's put in the `/opt/stackstorm`, but not symlinked into a PATH directory (e.g. /usr/local/bin) or similar.

I do think it would be a good idea to symlink it, but yeah, that's a simple st2-packages change I / we can do once this is merged.

**Comment by Kami** [comment]
Another place which needs to be updates is `st2common/setup.py`

**Comment by Kami** [comment]
I went ahead and merge changes into master - thanks!

I will make the setup.py and other change locally. Thanks again!

**Pull request merged**